polaris/viz/planar.py

```diff
@@ -137,8 +155,7 @@ def plot_horiz_field(ds, ds_mesh, field_name, out_file_name=None,  # noqa: C901
         cbar.set_label(cmap_title)
     if create_fig:
         plt.title(title)
-        plt.tight_layout(pad=0.5)
-        plt.savefig(out_file_name)
+        plt.savefig(out_file_name, bbox_inches='tight', pad_inches=0.2)
```

Comment:
For some reason, `plt.tigth_layout()` doesn't actually work well when saving figures to image files.  Instead, the combination of ` bbox_inches='tight'` and `pad_inches` (set to something pretty small) works well based on our experience in MPAS-Analysis.